scripts/classes/game_camera_2d.gd

```diff
@@ -4,6 +4,14 @@ extends Camera2D
 
 @export var default_resolution: Vector2 = Vector2(1920, 1080)
 
+@export var out_of_board_pixel_amount : float = 40
+
+var _default_scale : float = 1
+
+func update_scale(rows: int, pixelsize: int, gridsize: int):
+	var board_size = (pixelsize + gridsize) * rows + out_of_board_pixel_amount
+	_default_scale = default_resolution.y / board_size
+	_on_viewport_size_changed()
```

Comment:
since now we also call this method in update_scale, might be a good idea to rename ``_on_viewport_size_changed()`` to ``_update_zoom()``

Reply:
Agreed, The name has been updated